rpcs3/VKGSRender.vcxproj.filters

```diff
@@ -61,10 +61,11 @@
     </ClCompile>
     <ClCompile Include="Emu\RSX\VK\vkutils\image_helpers.cpp">
       <Filter>vkutils</Filter>
-    </ClCompile>
+    </ClCompile>
     <ClCompile Include="Emu\RSX\VK\VKOverlays.cpp" />
-    <ClCompile Include="Emu\RSX\VK\VKCompute.cpp" />
-    <ClCompile Include="Emu\RSX\VK\VKAsyncScheduler.cpp" />
+    <ClCompile Include="Emu\RSX\VK\VKCompute.cpp" />
+    <ClCompile Include="Emu\RSX\VK\VKAsyncScheduler.cpp" />

```

Comment:
suspicious

Reply:
visual studio works in mysterious ways

Reply:
The last person before must have done a manual edit.